tests/Behat/features/individual-block-validation.feature

```diff
@@ -72,17 +72,19 @@ Feature: Blocks are validated when inline saving individual blocks
     # Need to save the whole page to stop the alert
     And I press the "Save" button
 
-  Scenario: I can save a closed block after saving a page with a validation error
-    When I fill in "x" for "My Field" for block 1
-    And I press the "Save" button
-    # Page validation error
-    Then I should see "MyField cannot be x" in the ".alert.error" element
-    When I press the "View actions" button
-    And I click on the ".element-editor__actions-save" element
-    # Same validation error after page load with inline save
-    And I should see "MyField cannot be x" in the ".form__validation-message" element
-    Then I fill in "abc" for "My Field" for block 1
-    # Ensure react field is filled in before submitting
-    And I wait for 1 second
-    # Need to save the whole page to stop the alert
-    And I press the "Save" button
+# Test commented out due to to this issue
+# https://github.com/silverstripe/silverstripe-elemental/issues/1177
+#  Scenario: I can save a closed block after saving a page with a validation error
+#    When I fill in "x" for "My Field" for block 1
+#    And I press the "Save" button
+#    # Page validation error
+#    Then I should see "MyField cannot be x" in the ".alert.error" element
+#    When I press the "View actions" button
+#    And I click on the ".element-editor__actions-save" element
+#    # Same validation error after page load with inline save
+#    And I should see "MyField cannot be x" in the ".form__validation-message" element
+#    Then I fill in "abc" for "My Field" for block 1
+#    # Ensure react field is filled in before submitting
+#    And I wait for 1 second
+#    # Need to save the whole page to stop the alert
+#    And I press the "Save" button
```

Comment:
If this was passing before the refactor, it should be passing after the refactor instead of having that split as a separate issue.

Reply:
It was passing before, however only for the save button. The publish button is untested though if it was tested it would fail.  Therefore yes we are introducing a regression, though because it's related to an existing regression so I think it's correct commenting this out here

Note I spend about a day of time trying to fix this particular issue and I couldn't, so part of this is me deciding that I should quit trying to make this work and move on to more productive things. This regression is really a bit of an unimported edge case because it requires the user to first save the page, and then switch to inline saving which seems like a strange thing for a user to do. Also the inline form still submits and validation error still shows, it's just the user needs to manually open the element in order to see it

Reply:
Either remove the test completely, with a note to add in the same or equivalent test in the linked issue, or uncomment the test and make it work here. We don't include commented out code/tests in PRs.

> This regression is really a bit of a low importance edge case because it requires the user to first save the page, and then switch to inline saving which seems like a strange thing for a user to do.

I don't think that's strange at all.
Consider a content author editing a page. They may make changes in several places. But then a specific block has validation errors. So they fix that block, and save it on its own to make sure it passes validation before trying to save the page (and the rest of their content) again.

Nevertheless, the impact *is* quite small since it's just whether the block is open or not. It can be handled as a follow up issue, but not left commented out like this.

Reply:
Removed and added note to the linked issue for add the test back in